app/raid.js

```diff
@@ -46,7 +44,7 @@ class Raid {
 					// if start & end time do not exist, use creation time +X hours, to determine if raid should be removed
 					if (!end_time.isValid() && !start_time.isValid() && now > completion_time) {
 						raids_map.delete(raid_id);
-						return;
+
```

Comment:
Just curious, why remove the return?  I basically it it there so that once the raid was removed, you couldn't do anything further with it.  Now that it's gone, you COULD do more with it, but I can't see that as a good thing haha.

Reply:
I think it was just to appease IntelliJ, as it was sensing there was no logic past it anywhere in the forEach block and hence that return statement did nothing anyway, so it was calling it out as a redundant / unnecessary return.  You're right that logically it should probably be left there if we want to do anything else in that loop after the raid_map deletions though. 😛